data/json/furniture_and_terrain/furniture-storage.json

```diff
@@ -492,7 +492,12 @@
     "required_str": -1,
     "flags": [ "TRANSPARENT", "FLAMMABLE_HARD", "CONTAINER", "PLACE_ITEM", "MOUNTABLE" ],
     "deconstruct": {
-      "items": [ { "item": "2x4", "count": 2 }, { "item": "nail", "charges": [ 2, 5 ] }, { "item": "sheet_metal", "count": 1 } ]
+      "items": [
+        { "item": "2x4", "count": 1 },
```

Comment:
```suggestion
```
It's no longer a 2x4 when you cut it in half to create the base that goes on top of the wooden post to hold the mailbox.  I know you didn't create the original recipe.  It probably needs a 2x4s worth of wooden scraps which I guess are just splinters?